extension/src/triggers.rs

```diff
@@ -0,0 +1,56 @@
+use pgx::*;
+
+#[pg_extern]
+fn timescale_analytics_probe() {
+
+}
+
+extension_sql!(r#"
+CREATE OR REPLACE FUNCTION disallow_experimental_dependencies()
+  RETURNS event_trigger
+ LANGUAGE plpgsql
+  AS $$
+DECLARE
+  guc_set TEXT;
+  cmd record;
+  experimental_schema_id oid;
+BEGIN
+
+  guc_set := current_setting('timescale_analytics_install_experimental', true);
```

Comment:
This GUC doesn't match the one in the detail on the exception

Reply:
fixed